core/src/main/java/com/microsoft/applicationinsights/internal/quickpulse/DefaultQuickPulseDataSender.java

```diff
@@ -90,7 +90,7 @@ public void run() {
         } catch (Throwable t) {
             try {
                 stopped = true;
-                quickPulseStatus = QuickPulseStatus.ERROR;
+                quickPulseHeaderInfo = new QuickPulseHeaderInfo(QuickPulseStatus.QP_IS_ON);
```

Comment:
is this a change in behavior?

Reply:
Thank you for catching it! This was a mistake on my part